sda-commons-web-autoconfigure/src/main/java/org/sdase/commons/spring/boot/web/monitoring/SdaMonitoringConfiguration.java

```diff
@@ -7,9 +7,21 @@
  */
 package org.sdase.commons.spring.boot.web.monitoring;
 
+import org.sdase.commons.spring.boot.web.OpenPolicyAgentHealthIndicator;
+import org.springframework.boot.actuate.autoconfigure.health.ConditionalOnEnabledHealthIndicator;
 import org.springframework.boot.autoconfigure.AutoConfiguration;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.PropertySource;
 
 @AutoConfiguration
 @PropertySource("classpath:/org/sdase/commons/spring/boot/web/monitoring/monitoring.properties")
-public class SdaMonitoringConfiguration {}
+public class SdaMonitoringConfiguration {
+
+  @Bean
+  @ConditionalOnProperty(name = "opa.disable", havingValue = "false", matchIfMissing = true)
+  @ConditionalOnEnabledHealthIndicator("openPolicyAgent")
+  public OpenPolicyAgentHealthIndicator openPolicyAgentHealthContributor() {
+    return new OpenPolicyAgentHealthIndicator();
+  }
```

Comment:
But why would you want to create the bean if opa is disabled? Would not it consume more memory with a bean that would be never used? With the ` @ConditionalOnEnabledHealthIndicator` annotation you can even disable this bean even with the opa.disabled enabled, in case you do not want to perform  the health check

Reply:
I wonder why we should make it configurable to have the health check. As long as OPA is enabled, the health check should be active. Also [the docs](https://github.com/SDA-SE/sda-spring-boot-commons/pull/187/files#diff-2ead3b4ae5e0196eaed448f28100ca5495e172e2b954707c5c52ce2006f09f84R489) could be simplified to only _if OPA is enabled_. Then we don't need to bloat the docs with the defaults.

Reply:
I think we should provide configuration options to the user, so they can choose where having the health check or not. but thinking in this specific case it does not make sense to disable the health check, if the opa is enabled, because the pod will be marked as ready, but the requests will fail. So I removed this condition and left just the opa.disabled property check